sass/theme/wrbb-styles/dj-show-menu.scss

```diff
@@ -0,0 +1,3 @@
+.row .col-sm-3 img {
```

Comment:
This selector is a little sketch since it selects every `<img>` in a `col-sm-3`. There could be a scenario in the future where we want to add images to a `col-sm-3` but don't want this style applied. So I would suggest creating a new class for this style and adding it directly to the `<img>` tags in `dj-show-menu.php` and `home.php`. Also, I would put that style in `mainpage.scss`, then you wouldn't need this file.